src/generic/Poly.jl

```diff
@@ -39,6 +39,18 @@ base_ring(R::AbstractAlgebra.PolyRing{T}) where T <: RingElement = R.base_ring::
 """
 base_ring(a::PolynomialElem) = base_ring(parent(a))
 
+@doc Markdown.doc"""
+    change_base_ring(::Poly{T}, g::any)
+> Return the polynomial obtained by applying `g` to the coefficients. The new base ring is defined by the image of `1`.
+"""
+function change_base_ring(p::Poly{T}, g) where T <: RingElement
+   new_base_ring = parent(g(zero(base_ring(p.parent))))
```

Comment:
Not very important, but the doc says that the image of `1` is used, not that of `0`.